.gitignore

```
@@ -13,3 +13,4 @@
 .externalNativeBuild
 .cxx
 local.properties
+/app/src/main/java/br/com/elizandrakrier/myweatherapp/keys
```

Essa linha não funcionou, tenta deixar desse jeito aqui:
```suggestion
keys/
```

De acordo com o que vi, isso vai fazer com que qualquer pasta chamada keys no projeto seja ignorada.

[Link de referência](https://www.freecodecamp.org/news/gitignore-file-how-to-ignore-files-and-folders-in-git/)